### PR TITLE
[PE-4366] Updated Stencil to 1.8.9

### PR DESCRIPTION
**Pull request by janibolkvadze**
### Commit summary

- [x] Updated Stencil to 1.8.9
- [x] Fixed rerendering issue of Badge Custom element
- [x] Fixed rerendering issue of Button Custom element

@mattnickles @jllr @dani-cl-madrid 

https://ctl.atlassian.net/browse/PE-4366

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://e398522b9bcdbc62c24e3af8559df135f6235915-chi-pr.enterprise.ctl.io" target="_blank"> https://e398522b9bcdbc62c24e3af8559df135f6235915-chi-pr.enterprise.ctl.io </a>

File: src/custom-elements/src/components/badge/badge.tsx [hunk 1]
```diff
@@ -50,17 +50,14 @@ export class Badge {
...
-      this.slotBadgeContent = false;
-    }
-  }
-
+  
   componentWillLoad() {
     this.colorValidation(this.color);
     this.sizeValidation(this.size);
     this.variantValidation(this.variant);
+    if (!this.el.innerHTML.includes('<chi-icon')) {
```
Comment (dani-cl-madrid) [inline src/custom-elements/src/components/badge/badge.tsx]: Can you test if querySelector works better than `inerHTML.includes` function?

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/663/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://00aa669e91d403f8cb58aabdef829374ffc8de36-chi-pr.enterprise.ctl.io" target="_blank"> https://00aa669e91d403f8cb58aabdef829374ffc8de36-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/664/ for more details.

File: src/custom-elements/src/components/badge/badge.tsx (hunk 1 above)
Comment (janibolkvadze) [reply src/custom-elements/src/components/badge/badge.tsx]: Yes, querySelector works better, thanks!

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-o3cx9f, https://ops.elasticbox.com/services/instances/i-k4akgx

**Pull request merged**